src/client/sandbox/node/window.js

```diff
@@ -420,6 +420,28 @@ export default class WindowSandbox extends SandboxBase {
             };
         }
 
+        if (window.WebSocket) {
+            window.WebSocket = function (url, protocols) {
+                if (arguments.length === 0)
+                    return new nativeMethods.WebSocket();
+
+                const proxyUrl = getProxyUrl(url, { resourceType: stringifyResourceType({ isWebSocket: true }) });
+
+                if (arguments.length === 1)
+                    return new nativeMethods.WebSocket(proxyUrl);
+                else if (arguments.length === 2)
+                    return new nativeMethods.WebSocket(proxyUrl, protocols);
+
+                return new nativeMethods.WebSocket(proxyUrl, protocols, arguments[2]);
```

Comment:
Add test for this case. 
Rewrite as `WebSocket(arguments)` because constructor can be called as `new WebSocket(url, protocols, wrongParam1, wrongParam2, etc)`.